app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -986,6 +993,15 @@ class BrowserTabViewModel(
         }
     }
 
+    private suspend fun getBookmarkFolder(bookmark: SavedSite.Bookmark?): BookmarkFolder? {
```

Comment:
any reason method param should be nullable? if it's not a valid argument, I would remove nullability.

Reply:
instead of checking the nullability of the bookmark (of the saved site got from the bookmarksRepository) every time I use this function, I thought to add the null check inside the function, hence the parameter is nullable.

Will think of a better way.